app/serializer.py

```diff
@@ -0,0 +1,36 @@
+import json
+import xml.etree.ElementTree as ET
+from abc import ABC, abstractmethod
+
+from app.book import Book
+
+
+class AbstractSerializer(ABC):
+
+    @abstractmethod
+    def serialize(self) -> None:
+        pass
```

Comment:
```suggestion
class AbstractSerializer(ABC):
    @abstractmethod
    def serialize(self) -> None:
        pass
```
why you need empty line here?